pyside6_overlay/ui/components/rf4s_control_ui.py

```
@@ -0,0 +1,115 @@
+
+"""
+RF4S Control UI Component - Manages RF4S bot control interface
+"""
+
+from PySide6.QtWidgets import QWidget, QVBoxLayout, QHBoxLayout
```

Ruff (0.11.9)

`PySide6.QtWidgets.QWidget` imported but unused

Remove unused import: `PySide6.QtWidgets.QWidget`

(F401)